From fe897e08bd145ac5f96c62ad2e78c947d93c9b57 Mon Sep 17 00:00:00 2001
From: William <william@undefined.re>
Date: Fri, 23 Oct 2020 12:41:11 +0200
Subject: spell

-Subproject commit c2c21a3cf3bc42293efb65536f5b784efa66e5bf
+Subproject commit fe7889550dd0004d2ed5e3099f538eef47d8fa4b
diff --git a/modules/checkers/spell/config.el b/modules/checkers/spell/config.el
index 5886d79..b53d2ba 100644
--- a/modules/checkers/spell/config.el
+++ b/modules/checkers/spell/config.el
@@ -129,12 +129,12 @@
                      conf-mode-hook
                      prog-mode-hook)
                    #'spell-fu-mode))
-      :config
-      (map! :after spell-fu
-            :map override
-            :n [return]
-            (cmds! (memq 'spell-fu-incorrect-face (face-at-point nil t))
-                   #'+spell/correct))
+      ;; :config
+      ;; (map! :after spell-fu
+      ;;       :map override
+      ;;       :n [return]
+      ;;       (cmds! (memq 'spell-fu-incorrect-face (face-at-point nil t))
+      ;;              #'+spell/correct))

       (defadvice! +spell--create-word-dict-a (_word words-file _action)
         :before #'spell-fu--word-add-or-remove
--
2.29.0
